<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetGeometry</class>
 <widget class="QWidget" name="WidgetGeometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑网格</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>根骨骼</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="ancestorBone"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>网格名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>面数量</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>顶点数量</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="numVertex">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="numFace">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>脚本</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QPlainTextEdit" name="script"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="type">
     <item>
      <property name="text">
       <string>固定颜色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>固定方向</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>地面</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>变换颜色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>碰撞箱</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>网格类型</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>所用材质</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="materialId"/>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="alwaysStand">
       <property name="text">
        <string>始终向上</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="verticalGround">
       <property name="text">
        <string>垂直地面</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>name</zorder>
  <zorder>ancestorBone</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_4</zorder>
  <zorder>numVertex</zorder>
  <zorder>label_5</zorder>
  <zorder>numFace</zorder>
  <zorder>script</zorder>
  <zorder>label_3</zorder>
  <zorder>materialId</zorder>
  <zorder>label_7</zorder>
  <zorder>type</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
